test/common/tests.go

```diff
@@ -27,8 +27,8 @@ var (
 		{"E01 - Verify Grafana Route is accessible", TestGrafanaExternalRouteAccessible},
 		{"E05 - Verify Grafana Route returns dashboards", TestGrafanaExternalRouteDashboardExist},
 		{"C04 - Verify Alerts exist", TestIntegreatlyAlertsExist},
-		{"C01 - Verify Alerts are not firing apart from DeadMansSwitch", TestIntegreatlyAlertsFiring},
-		{"C01.1 - Verify Alerts are not firing apart from DeadMansSwitch", TestIntegreatlyAlertsPendingOrFiring},
+		{"Verify Alerts are not firing during or after intsallation apart from DeadMansSwitch", TestIntegreatlyAlertsFiring},
```

Comment:
@R-Lawton Do we need to have this test associated with some kind of test ID, all others are? @tonyxrmdavidson @damienomurchu 

Also typo `intsallation` -> `installation`

Reply:
i wasnt sure as there's some tests in "ALL TESTS" that have no id